industrial_ci/src/workspace.sh

```diff
@@ -303,6 +303,12 @@ function ici_test_workspace {
     local extend=$1; shift
     local ws=$1; shift
 
-    ici_run "run_${name}_test" builder_run_tests "$extend" "$ws"
+    # Python coverage only support colcon builder in ROS2
```

Comment:
I would move this into  builder_run_tests somehow.
Not sure how to handle the target yet..

Reply:
One option can be to check `"$target_ws" = ~/target_ws`, I wouldn't say it will be pretty tho.

Reply:
I will try to come up with a proposal later